migration-tool/src/main/resources/generate-recipes

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+
+from scripts.generate_upgrade_sdk_dependencies_recipe import generate_upgrade_sdk_dependencies_recipe
```

Comment:
Id it possible to write this in Java instead of python ?

Reply:
It's certainly possible, but since Java is quite verbose, I find it easier to use python for this use-case. We use python elsewhere in this repo for other scripts, so this is not a new dependency.